include/IGPUMappedBuffer.h

```diff
@@ -17,10 +17,13 @@ namespace video
 class IGPUMappedBuffer : public virtual video::IGPUBuffer
 {
     public:
-        //! WARNING: RESIZE will invalidate pointer
-        //! WARNING: NEED TO FENCE BEFORE USE!!!!!!!!!!!!!
+		//! Gets internal pointer.
+        /** WARNING: RESIZE will invalidate pointer!
+        WARNING: NEED TO FENCE BEFORE USE!
+		@returns Internal pointer. */
         virtual void* getPointer() = 0;
 
+		//! @returns Always true.
```

Comment:
We only support persistently mapped buffers with ARB_buffer_storage.
Its almost the fastest across all cards, and is more in line with what Vulkan has.
Please don't ask us to support Buffer Orpahning, or Map/Unmap.